ui/apps/akasha/src/extensions/beam-editor/beam-editor.tsx

```diff
@@ -66,6 +70,18 @@ export const BeamEditor: React.FC = () => {
     }
   }, [blocksInUse]);
 
+  React.useEffect(() => {
```

Comment:
why not do this checks in the addTag function?

Reply:
Alright. Let me check also in the keyup handler